server/api.js

```diff
@@ -1,82 +1,109 @@
-
 import { Router } from "express";
-
 import { Connection } from "./db";
-
 const router = new Router();
-
 router.get("/", (_, res, next) => {
-
-	Connection.connect((err) => {
-		if (err) {
-			return next(err);
-		}
-		res.json({ message: "Hello, world!" });
-	});
+  Connection.connect((err) => {
```

Comment:
```suggestion
 connection.connect((err) => {
```
Make this `connection` variable with a lower case letter at the start